gnosis_vpn/Cargo.toml

```diff
@@ -21,4 +21,3 @@ url.workspace = true
 # Target-specific dependencies for memory allocators
 [target.'cfg(target_os = "linux")'.dependencies]
 mimalloc = { workspace = true }
-tikv-jemallocator = { workspace = true }
```

Comment:
Ideally you should implement everything using the jemalloc, since it is much more efficient with hopr-lib and offers a significant throughput increase.